demo/full/scripts/modules/ChartData.js

```diff
@@ -8,7 +8,7 @@ export default ({ state }, { maxSize }) => {
         data.splice(0, (data.length + 1) - maxSize);
       }
       data.push({
-        date: Date.now(),
+        date: performance.now(),
```

Comment:
No real impact here.
I just found that relying on `performance.now()` was more logical.